tasks/install_os.yml

```diff
@@ -10,25 +10,20 @@
   with_together:
     - [ "{{ hypervisor_password }}", 'stack']
     - ['root', 'stack']
-
   ignore_errors: true
 
+# forman is not having RHEL 7.9 and 8.4. So we need to install previsous version and upgrade later
 - name: set os_install when installed OS version not matching with the needed
   vars:
     rhel_stdout: "{{ (rhel_release.results[0].stdout) | ternary(rhel_release.results[0].stdout, rhel_release.results[1].stdout) }}"
   set_fact:
-    os_install: "{{ needed_os }}"
-  when: rhel_stdout == "" or (needed_os.split()[1] not in rhel_stdout) or (os_search_string not in rhel_stdout)
-
-- name: set os_install before updating os RHEL 7.9
+    os_install: "{{ (needed_os == 'RHEL 8.4') | ternary('RHEL 8.2', ((needed_os == 'RHEL 7.9')| ternary('RHEL 7.8', needed_os))) }}"
+  when: (force_reprovision == true) or (rhel_stdout == "" or (needed_os.split()[1] not in rhel_stdout) or (os_search_string not in rhel_stdout))
+    
```

Comment:
linter is failing due to trailing space on line 22 https://github.com/redhat-performance/jetpack/runs/4332652570?check_suite_focus=true#step:4:97